rel/runtime_config.exs

```diff
@@ -0,0 +1,13 @@
+use Mix.Config
+
+cert = System.gen_env("DEV_CERT_PEM")
+if cert == nil do
+  raise "Your developer certificate is not set correctly please ensure you set it to the environment variable $DEV_CERT_PEM"
+end
+
+if File.exists?(cert) == false do
+  raise "The DEV_CERT_PEM environment variable is set, however the file path is incorrect"
+end
+
+config :mozart_fetcher,
+        Keyword.new([{String.to_atom(String.downcase("DEV_CERT_PEM")), System.get_env("DEV_CERT_PEM")}])
```

Comment:
does this not work then? 🤔 
```suggestion
        Keyword.new([:dev_cert_pem, System.get_env("DEV_CERT_PEM")}])
```

Reply:
Sorry copy pasted that bit from belfrage one